scripts/haas_network.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
```

Comment:
Originally I had also added the dependency for Http server such that if someone tries to start this service it makes sure that http server is running before this service is invoked.

But @henn argued that we may want to run the web server and the network server on different machines in future hence such dependency is not required. 

On the same line of reasoning, do you think we will ever run the haas_network and the database on separate machines. If the answer is no, I will add database dependency to the service. 

Since we support only postgres for production, right now will put only that. 
With a comment in that it can support more databases in future. 

The dev service can support both postgres and sqlite.